### PR TITLE
Simplify kts

### PR DESCRIPTION
**Pull request by Goooler**
- Replace deprecated methods.
- Merge `allprojects`.

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -48,43 +48,7 @@ allprojects {
...
-  tasks.withType<Test> {
-    testLogging {
-      events(STARTED, PASSED, SKIPPED, FAILED)
-      exceptionFormat = TestExceptionFormat.FULL
-      showStandardStreams = false
-    }
-  }
-}
 
-allprojects {
```
Comment (Egorand) [inline build.gradle.kts:87]: Does it make any difference that this `allprojects` is at the top level of the script, while the one you merged it into is inside the `buildscript` block?

Comment (Goooler) [reply build.gradle.kts:87]: `allprojects` is still at the top level.

image

Comment (Egorand) [reply build.gradle.kts:87]: Got it, sorry, I misread the diff!

Comment (Goooler) [reply build.gradle.kts:87]: Don't worry.

File: okio-nodefilesystem/build.gradle.kts [hunk 2]
```diff
@@ -29,7 +29,7 @@ kotlin {
   }
   sourceSets {
     all {
-      languageSettings.useExperimentalAnnotation("kotlin.RequiresOptIn")
+      languageSettings.optIn("kotlin.RequiresOptIn")
```
Comment (swankjesse) [inline okio-nodefilesystem/build.gradle.kts:32]: ooooh neat

**Review by swankjesse — APPROVED** [review]
Thank you!

**Pull request merged**